### DISCUSSION TITLE
Some question of data diagnosis

### DISCUSSION THREAD

**Discussion by qiudashuai1** [CoBrALab/RABIES#393] [General]
Hi, I have running steps of preprocessing, cofounding correction, and data diagnosis in following code:
sudo docker run -it --rm --user $(id -u) \
-v $PWD/input_BIDS:/input_BIDS:ro \
-v $PWD/preprocess_outputs:/preprocess_outputs/ \
ghcr.io/cobralab/rabies:latest -p MultiProc preprocess /input_BIDS/ /preprocess_outputs/ --apply_STC --TR 2 --bold_autobox --commonspace_reg  masking=true,brain_extraction=false,template_registration=SyN,fast_commonspace=false

sudo docker run -it --rm --user $(id -u) \
-v $PWD/input_BIDS:/input_BIDS:ro \
-v $PWD/preprocess_outputs:/preprocess_outputs/ \
-v $PWD/confound_correction_outputs:/confound_correction_outputs/ \
ghcr.io/cobralab/rabies:latest -p MultiProc confound_correction /preprocess_outputs/ /confound_correction_outputs/ --conf_list WM_signal CSF_signal vascular_signal mot_6 --smoothing_filter 0.3 

sudo docker run -it --rm --user $(id -u) \
-v $PWD/input_BIDS:/input_BIDS:ro \
-v $PWD/preprocess_outputs:/preprocess_outputs/ \
-v $PWD/confound_correction_outputs:/confound_correction_outputs/ \
-v $PWD/analysis_outputs:/analysis_outputs/ \
ghcr.io/cobralab/rabies:latest -p MultiProc analysis /confound_correction_outputs /analysis_outputs/ \
--prior_maps "/home/mambauser/.local/share/rabies/melodic_IC.nii.gz" \
--data_diagnosis

But the spatial diagnosis seems according with absent category:
![sub-con1_task-rest_bold_spatial_diagnosis](https://github.com/user-attachments/assets/4ed9b892-7892-4883-ae8f-0f39a02edc15)
The temporal diagnosis seems fine with no obvuous spike:
![sub-con1_task-rest_bold_temporal_diagnosis](https://github.com/user-attachments/assets/e27c65bd-cb2c-4278-9437-337f3d2335c6)
However, the dice index lower than 0.4, what should I do to improve the quality?
![DR0_sample_distribution](https://github.com/user-attachments/assets/93e33eff-6a1e-4385-8ab7-ac2a9b3dbed5)
![DR0_QC_maps](https://github.com/user-attachments/assets/e00d41b2-bbe7-42d9-8df3-a08fd92f91a7)

**Comment by gdevenyi** [comment]
> But the spatial diagnosis seems according with absent category:

Can you please rephrase this as a question.

**Reply by qiudashuai1** [reply]
Question 1: The temporal diagnosis seems fine with no obvuous spike. However, network map in spatial diagnosis seems according with absent category. What should I do to improve it?
Question 2:the dice index lower than 0.4, what should I do to improve the quality?

**Comment by Gab-D-G** [comment]
Hi @qiudashuai1, you correctly assessed that your dataset has low network detection. The fact that it is absent in the spatial diagnosis and has low Dice index are essentially the same thing. I cannot tell you necessarily what the cause is, but the most common culprit in my experience is that anesthesia levels are too high, thus inhibiting network activity. Another reason can be a faulty scanner sequence. This is something that can only be properly tested at acquisition.
However, I can recommend 2 more things to try with RABIES first: 1-do a group-ICA across many scans, and see whether you can find some networks. That will tell you whether any network activity can be found across the entire dataset. 2-You may try aCompCor_5 or --image_scaling voxelwise_standardization. These are two techniques that can in some instance enhance network detection in my experience. 
I hope this helps!
